<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Contacts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Contacts</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" rowspan="4" colspan="2">
            <widget class="QListWidget" name="m_ips"/>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLineEdit" name="m_name"/>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Phone</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="m_phone"/>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="m_addr"/>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>address</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>extra</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QTextEdit" name="m_extra"/>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>List of connected client</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="5" rowspan="3">
            <widget class="QListWidget" name="m_connectedip"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>248</width>
          <height>205</height>
         </rect>
        </property>
        <property name="title">
         <string>Voice Over IP</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_cur_ip">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="text">
            <string>420</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="text">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>ssl</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>simple</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rsa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>xor</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Encryption</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>key</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>pitch shifting</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_pitch">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>230</y>
          <width>48</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-6</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbTheme">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>469</width>
          <height>36</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove">
        <property name="text">
         <string>remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="serverbtn">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>not designed for use with huds or voice recognition software mainly leftover ai's inside space suit huds</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
